src/main/kotlin/org/rust/cargo/runconfig/ui/CargoCommandConfigurationEditor.kt

```diff
@@ -91,6 +93,13 @@ class CargoCommandConfigurationEditor(project: Project)
     private val requiredFeatures = CheckBox("Implicitly add required features if possible", true)
     private val allFeatures = CheckBox("Use all features in tests", false)
     private val emulateTerminal = CheckBox("Emulate terminal in output console", false)
+    private val withSudo = CheckBox(
+        if (SystemInfo.isWindows) "Run with Administrator privileges" else "Run with root privileges",
+        false
+    ).apply {
+        // TODO: remove when `com.intellij.execution.process.ElevationService` supports error stream redirection
+        isEnabled = isFeatureEnabled(RsExperiments.BUILD_TOOL_WINDOW)
```

Comment:
Let's also show warning in run configuration editor about sudo in tests. See `org.rust.cargo.runconfig.command.CargoCommandConfiguration#checkConfiguration`, `org.rust.cargo.runconfig.command.CargoCommandConfiguration#showTestToolWindow` and `com.intellij.execution.configurations.RuntimeConfigurationWarning`

Reply:
https://github.com/intellij-rust/intellij-rust/pull/7113#issuecomment-856655170